### ISSUE TITLE
Multiple warnings when server is running in Docker container

### ISSUE THREAD

**Issue by Beefycoder** [RuntimeTools/SwiftMetrics#212]
When running Kitura from a Docker container I see multiple warnings appearing periodically as the server is running.  I don't see this error when running in Xcode, just when I start the app via the `kitura run ` cli command.

```
[2019-03-26T21:28:19.004Z] [INFO] [HTTPServer.swift:195 listen(on:)] Listening on port 8080
[Tue Mar 26 21:28:50 2019] com.ibm.diagnostics.healthcenter.loader WARNING: formatCPU(): Could not obtain/parse CPU usage data.
[Tue Mar 26 21:28:50 2019] com.ibm.diagnostics.healthcenter.loader WARNING: formatCPU(): Could not obtain/parse CPU usage data.
[Tue Mar 26 21:29:06 2019] com.ibm.diagnostics.healthcenter.loader WARNING: formatCPU(): Could not obtain/parse CPU usage data.
[Tue Mar 26 21:29:06 2019] com.ibm.diagnostics.healthcenter.loader WARNING: formatCPU(): Could not obtain/parse CPU usage data.
[Tue Mar 26 21:29:20 2019] com.ibm.diagnostics.healthcenter.loader WARNING: formatCPU(): Could not obtain/parse CPU usage data.
[Tue Mar 26 21:29:20 2019] com.ibm.diagnostics.healthcenter.loader WARNING: formatCPU(): Could not obtain/parse CPU usage data.
[Tue Mar 26 21:31:45 2019] com.ibm.diagnostics.healthcenter.loader WARNING: formatCPU(): Could not obtain/parse CPU usage data.
[Tue Mar 26 21:31:45 2019] com.ibm.diagnostics.healthcenter.loader WARNING: formatCPU(): Could not obtain/parse CPU usage data.
[Tue Mar 26 21:32:19 2019] com.ibm.diagnostics.healthcenter.loader WARNING: formatCPU(): Could not obtain/parse CPU usage data.
[Tue Mar 26 21:32:19 2019] com.ibm.diagnostics.healthcenter.loader WARNING: formatCPU(): Could not obtain/parse CPU usage data.
[Tue Mar 26 21:32:45 2019] com.ibm.diagnostics.healthcenter.loader WARNING: formatCPU(): Could not obtain/parse CPU usage data.
[Tue Mar 26 21:32:45 2019] com.ibm.diagnostics.healthcenter.loader WARNING: formatCPU(): Could not obtain/parse CPU usage data.
[Tue Mar 26 21:32:55 2019] com.ibm.diagnostics.healthcenter.loader WARNING: formatCPU(): Could not obtain/parse CPU usage data.
[Tue Mar 26 21:32:55 2019] com.ibm.diagnostics.healthcenter.loader WARNING: formatCPU(): Could not obtain/parse CPU usage data.
[Tue Mar 26 21:33:33 2019] com.ibm.diagnostics.healthcenter.loader WARNING: formatCPU(): Could not obtain/parse CPU usage data.
[Tue Mar 26 21:33:33 2019] com.ibm.diagnostics.healthcenter.loader WARNING: formatCPU(): Could not obtain/parse CPU usage data.
[Tue Mar 26 21:34:03 2019] com.ibm.diagnostics.healthcenter.loader WARNING: formatCPU(): Could not obtain/parse CPU usage data.
[Tue Mar 26 21:34:03 2019] com.ibm.diagnostics.healthcenter.loader WARNING: formatCPU(): Could not obtain/parse CPU usage data.
[Tue Mar 26 21:34:15 2019] com.ibm.diagnostics.healthcenter.loader WARNING: formatCPU(): Could not obtain/parse CPU usage data.
[Tue Mar 26 21:34:15 2019] com.ibm.diagnostics.healthcenter.loader WARNING: formatCPU(): Could not obtain/parse CPU usage data.
[Tue Mar 26 21:34:27 2019] com.ibm.diagnostics.healthcenter.loader WARNING: formatCPU(): Could not obtain/parse CPU usage data.
[Tue Mar 26 21:34:27 2019] com.ibm.diagnostics.healthcenter.loader WARNING: formatCPU(): Could not obtain/parse CPU usage data.
[Tue Mar 26 21:35:13 2019] com.ibm.diagnostics.healthcenter.loader WARNING: formatCPU(): Could not obtain/parse CPU usage data.
[Tue Mar 26 21:35:13 2019] com.ibm.diagnostics.healthcenter.loader WARNING: formatCPU(): Could not obtain/parse CPU usage data.
[Tue Mar 26 21:35:39 2019] com.ibm.diagnostics.healthcenter.loader WARNING: formatCPU(): Could not obtain/parse CPU usage data.
[Tue Mar 26 21:35:39 2019] com.ibm.diagnostics.healthcenter.loader WARNING: formatCPU(): Could not obtain/parse CPU usage data.
```

**Comment by schlingding** [comment]
This also occurs in Cloud Foundry

**Comment by schlingding** [comment]
https://github.com/RuntimeTools/SwiftMetrics/pull/218

**Comment by ianpartridge** [comment]
Fixed in #218 and released in `2.6.3`.
